src/vs/workbench/contrib/chat/test/common/chatWordCounter.test.ts

```diff
@@ -22,9 +22,20 @@ suite('ChatWordCounter', () => {
 			['hello', 1, 'hello'],
 			['hello world', 0, ''],
 			['here\'s, some.   punctuation?', 3, 'here\'s, some.   punctuation?'],
-			['| markdown | _table_ | header |', 3, '| markdown | _table_ | header'],
+			['| markdown | _table_ | header |', 3, '| markdown | _table_ | header |'],
```

Comment:
I tweaked the logic so that if we request all words in the string, we now return the entire string, including the trailing non-word suffix

Previously I believe it only returned up to the end of the last word